account/account_test.go

```diff
@@ -883,11 +887,11 @@ func TestTransactionHashdeployAccount(t *testing.T) {
 					},
 				},
 				SenderAddress: utils.TestHexToFelt(t, "0x88d0038623a89bf853c70ea68b1062ccf32b094d1d7e5f924cda8404dc73e1"),
-				ExpectedHash:  utils.TestHexToFelt(t, "0x5b6b5927cd70ad7a80efdbe898244525871875c76540b239f6730118598b9cb"),
+				ExpectedHash:  utils.TestHexToFelt(t, "0x41ffcaa2f472e58b496f58aabae0590d4b2cf6117201c988d51c3f040e489e6"),
```

Comment:
This also doesn't seem to exist on the sepolia network.

Reply:
Didn't find this line in the code. Hence, considered fixed.